<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>correctWindow</class>
 <widget class="QWidget" name="correctWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>230</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>тематика</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QComboBox" name="startMatter"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>жанр</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="3">
     <widget class="QComboBox" name="startGenre"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>автор</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QLineEdit" name="startAuthor"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>название</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>цена</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>перевод</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="4">
     <widget class="QLineEdit" name="startTranslate"/>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QLabel" name="label7">
      <property name="text">
       <string>время чтения</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLineEdit" name="startTime"/>
    </item>
    <item row="4" column="2" colspan="4">
     <widget class="QLineEdit" name="startPrice"/>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QLineEdit" name="startName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
